packages/core/import-condition-proxy.mjs

```diff
@@ -0,0 +1,4 @@
+export * from './dist/redux-saga-core.cjs.js'
```

Comment:
I want to avoid [the dual package hazard](https://nodejs.org/api/packages.html#dual-package-hazard) and this wrapper acts as an ES module wrapper, like the one described [here](https://nodejs.org/api/packages.html#approach-1-use-an-es-module-wrapper)

You can import CJS packages in node. So in theory we could just not provide the `import` condition at all. However, there is a super annoying issue in node.js when it comes to default imports for CJS modules - they always resolve to the whole "namespace" (`module.exports`) and not to the actual `module.exports.default`. So this file just imports the CJS module (as it is allowed) and "adapts" its shape to the one that we actually document in the docs etc.